packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -879,7 +878,7 @@ class TimelineEvent {
   @visibleForTesting
   TimelineEvent deepCopy() {
     final copy = TimelineEvent(traceEvents.first);
-    copy.endTime = endTime;
+    copy.timeRange.end = timeRange.end;
```

Comment:
A bit odd the deepCopy doesn't have the startTime of the item being copied.

Reply:
The start time is set in the TimelineEvent constructor because the timestamp is included in the trace event.